CHANGELOG.md

```diff
@@ -49,6 +50,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking
 
+* (04-channel) [\#220](https://github.com/cosmos/ibc-go/pull/220) Channel legacy handler functions were removed. Please use the MsgServer functions or directly call the channel keeper's handshake function.
```

Comment:
technically we shouldn't do API breaking changes unless absolutely necessary since we are in beta, but I think this is an ok exception since this is non state machine breaking and the handler functions shouldn't be used outside of ibc-go. If we were on an rc I wouldn't include it